app/views/rankings/show.html.erb

```diff
@@ -1,5 +1,30 @@
 <%= render 'layouts/navbar' %>
 <p id="notice"><%= notice %></p>
 
-<%= link_to 'Edit', edit_ranking_path(@ranking) %> |
-<%= link_to 'Back', rankings_path %>
+<div class="mx-4 my-4">
+  <h2>Rankings</h2>
+
+  <table style="width:65%">
+    <thead>
+    <tr>
+      <th>Time</th>
+      <th>Rank</th>
+    </tr>
+    </thead>
+
+    <tbody>
+    <% @rankings.each do |ranking| %>
+      <tr>
+<!--        <td> <%#=@times.find(ranking.project_time_id).date_time.to_formatted_s(:long)%></td>-->
```

Comment:
what was the issue here?
each ranking owns a project_time via project_time_id column (in ranking model, there is a `belongs_to project_time`, so have you tried `ranking.project_time.date_time....`?